plugins/scaffolder-backend/src/scaffolder/actions/builtin/catalog/write.ts

```diff
@@ -28,6 +28,11 @@ export function createCatalogWriteAction() {
       input: {
         type: 'object',
         properties: {
+          name: {
```

Comment:
Wonder if we should call it `path` or `filePath` or something like that instead? Since that's effectively what it ends up being

Reply:
makes sense! I used `name` since I saw that param was already defined but never implemented (I assumed it was originally intended to support custom filenames but forgotten?). Updated!